### PR TITLE
header in "about" for an asterisk

### PR DESCRIPTION
**Pull request by DavidTereshchenko**
сам список nav не робив з grid, думаю він там лишній

### CODE REVIEW

**Comment by vladyslav-yermolin-moc** [comment]
https://github.com/MastersAcademy/fe-course-basic-2023/pull/115
Assigning @MZabolotnev, as he has checked the part without `*`

**Comment by DavidTereshchenko** [comment]
also i added a filter which also has grid

File: homeworks/david.tereshchenko_DavidTereshchenko/GameProject/css/style.css [hunk 1]
```diff
@@ -103,9 +104,10 @@ body {
 }
 
 .header__body {
-  display: flex;
+  display: flex; /* на мою жумку тут не потрібно робити грід */
```
Comment (MZabolotnev) [inline homeworks/david.tereshchenko_DavidTereshchenko/GameProject/css/style.css:107]: Comments in the code are not for communication. You can comment to separate blocks of code by meaning, or TODO for the future. Please remove this

**Review by MZabolotnev — CHANGES_REQUESTED** [review]

File: homeworks/david.tereshchenko_DavidTereshchenko/GameProject/css/style.css (hunk 1 above)
Comment (DavidTereshchenko) [reply homeworks/david.tereshchenko_DavidTereshchenko/GameProject/css/style.css]: oou, sorry, forgot this

**Review by MZabolotnev — APPROVED** [review]

**Pull request merged**